### ISSUE TITLE
Postfix policyd-spf issuing traceback on incoming mail, mails being falsely rejected

### ISSUE THREAD

**Issue by petabyteboy** [NixOS/nixpkgs#72791]
**Describe the bug**
Some emails on my postfix mailserver are being rejected due to the policyd-spf service crashing when processing the incoming mail.

**To Reproduce**
1. Set up a mail server with https://gitlab.com/simple-nixos-mailserver/nixos-mailserver
2. Receive mail from vultr.com

**Expected behavior**
The mail should be accepted. I would not expect to see a python traceback in my logs.

**Screenshots**
https://termbin.com/ylzy

**Additional context**
I think this bug reported on the debian mailing list is related:
https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1701553.html

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.5, NixOS, 20.03pre200231.7827d3f4497 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre200231.7827d3f4497"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: postfix
```

**Comment by petabyteboy** [comment]
This is a bug introduced with pyspf 2.0.13, and it will be fixed in 2.0.14.
https://github.com/sdgathman/pyspf/issues/17

**Comment by petabyteboy** [comment]
I suggest we revert https://github.com/NixOS/nixpkgs/pull/70733 until the fix is released.
CC @FRidh
